src/slackbot/accountSummaryCommandHandler.ts

```diff
@@ -32,6 +33,11 @@ export async function accountSummary(accountId: string) {
 
   const failingChecksGoals = await summarizeChecksGoal(checksWithChangePoints);
 
+  const errorPatterns = await findErrorClustersForChecks(
+    accountSummary.checks.map((c) => c.id),
```

Comment:
can this be empty?

Reply:
added some code which handles this further up now